Dockerfile

```diff
@@ -11,8 +21,10 @@ RUN yum install -y python-requests && \
     yum clean all && \
     rm -rf /var/cache/{yum,dnf}/*
 
-RUN mkdir /tftpboot && \
-    cp /usr/share/ipxe/undionly.kpxe /usr/share/ipxe/ipxe.efi /tftpboot/
```

Comment:
Let's remove `ipxe-bootimgs` from the list of installed packages above?

Reply:
Done